controllers/cloudinit/scripts/00-configure-snapstore-proxy.sh

```diff
@@ -18,12 +23,12 @@ if ! type -P curl ; then
   done
 fi
 
-while ! curl -sL http://"${1}"/v2/auth/store/assertions | snap ack /dev/stdin ; do
+while ! curl -sL "${1}"://"${2}"/v2/auth/store/assertions | snap ack /dev/stdin ; do
```

Comment:
we want to preserve backwards compatibility. `${1}` comes directly from `input.SnapstoreProxyScheme`, but that might be unset, in which case this breaks. Make sure you fallback to http if that is empty (prefer doing the fallback in the Go code, not the bash script)